config.json

```diff
@@ -112,6 +112,10 @@
       },
       {
         "advisory": "https://github.com/advisories/GHSA-4q6p-r6v2-jvc5"
+      },
+      {
+        "advisory": "https://github.com/advisories/GHSA-7fh5-64p2-3v2j",
+        "issue": "https://github.com/brave/brave-browser/issues/33384"
```

Comment:
this is not really a ReDoS as far as i can tell so maybe the comment should be updated

Reply:
Ah I didn't notice that, can we add comments here inside the {advisory,issue} object without anything breaking?

Reply:
probably yes but @wknapik can confirm